netbox_librenms_plugin/templates/netbox_librenms_plugin/htmx/device_validation_details.html

```
@@ -525,21 +609,373 @@ <h6 class="card-header d-flex justify-content-between align-items-center">
+          <form hx-post="{% url 'plugins:netbox_librenms_plugin:device_add_as_oob' device_id=libre_device.device_id %}"
+                hx-swap="none">
+            {% csrf_token %}
+            <input type="hidden" name="existing_device_id" value="{{ validation.existing_device.pk }}">
+            {% include "netbox_librenms_plugin/inc/_hidden_server_key.html" %}
+            {% include "netbox_librenms_plugin/htmx/_oob_interface_select.html" %}
+            <button type="submit" class="btn btn-sm btn-primary">
+              <i class="mdi mdi-chip"></i> Add as OOB to {{ validation.existing_device.name }}
+            </button>
+          </form>
+        </div>
+      </div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)